src/uct/ib/base/ib_device.h

```diff
@@ -238,6 +238,12 @@ typedef struct uct_ib_device {
     uint8_t                     pci_cswap_arg_sizes;
     uint8_t                     atomic_align;
     uint8_t                     lag_level;
+    uint8_t                     req_notify_cq_support;
+    uint8_t                     ordered_send_comp;
+    uint64_t                    mr_access_flags;
+    uint16_t                    max_sq_sge;       /* Send queue SGE count */
+    uint32_t                    max_inline_data;  /* Max inline buffer bytes */
```

Comment:
imo the names are already self-explanatory

Reply:
fixed